pages/about/groups/task-forces/wcag2ict/work-statement.md

```diff
@@ -24,113 +24,127 @@ The WCAG2ICT Task Force is a task force of the [Accessibility Guidelines Working
+The WCAG2ICT Phase 1 objectives have been completed by incorporating guidance for WCAG 2.1 and 2.2 guidelines, Level A and AA success criteria, and their associated glossary definitions. Phase 1 of WCAG2ICT also added new content that identifies WCAG success criteria that are problematic when applied to some non-Web ICT. The 2024 [Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)](https://www.w3.org/TR/wcag2ict-22/) was published as a result. 
+
+This work statement sets further objectives for Phase 2 of the WCAG2ICT Note updates as there have been concerns with determining which success criteria can (or should) apply for a particular technology as well as the possibility of suggesting normative language changes to some success criteria to make them more directly applicable in non-web contexts (See the scope of work section for further details and explanation). 
+
+In addition, the task force would like to develop content to address misunderstandings of what the WCAG2ICT Note does and does not provide which we hope will address incorrect assumptions and provide an explanation of problems that the WCAG2ICT Note does not solve. How this content will be delivered (as a separate document, web page, or within the WCAG2ICT Note) will be determined once the key content has been developed.
 
 ## Scope of Work
 
 The work of the task force includes:
 
 *   Determining how to apply each of the new WCAG 2.1 and WCAG 2.2 (hereafter called “WCAG 2.x") guidelines and Levels A, AA, and AAA success criteria to non-Web ICT; this includes provision of additional details for application to closed product software and where applying individual criteria might be difficult and why; and
 *   Producing an updated Working Group Note (a form of W3C Technical Report)—following the W3C process under the coordination, review, and approval of the AG WG—that describes:
-    *   how each of the new WCAG 2.x A, AA, and AAA success criteria could apply to non-Web ICT;
-    *   how each of the new WCAG 2.x guidelines could apply to non-Web ICT;
-    *   clarification of challenges of applying particular WCAG 2.x success criteria to non-web ICT, including closed product software; and
-    *   address challenges in [Open issues labeled WCAG2ICT](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue+is%3Aopen+label%3Awcag2ict).
+    *   how each of the new WCAG 2.x A and AA success criteria could apply to non-Web ICT (completed in Phase 1)
+    *   how each of the new WCAG 2.x guidelines could apply to non-Web ICT (completed in Phase 1)
+    *   clarification of challenges of applying particular WCAG 2.x success criteria to non-web ICT, including closed product software (completed in Phase 1)
+    *   address challenges in [Open issues labeled WCAG2ICT](https://github.com/w3c/wcag2ict/issues?q=is%3Aissue+is%3Aopen+label%3Awcag2ict) (completed in Phase 1)
```

Comment:
> https://github.com/w3c/wcag2ict/issues?q=is%3Aissue+is%3Aopen+label%3Awcag2ict

@daniel-montalvo This link leads to the following warning message on GitHub: "Filter contains 1 issue: Invalid value wcag2ict for label"

Reply:
Thanks for pointing this out @remibetin .

I am deferring this to @maryjom -- I think better to just point to all [Closed issues addressed in phase 1](https://github.com/w3c/wcag2ict/issues/?q=is%3Aissue%20state%3Aclosed), but I'd like to get her thoughts on this.

Reply:
```suggestion
    *   address challenges in issues. [See closed issues addressed in Phase 1](https://github.com/w3c/wcag2ict/issues/?q=is%3Aissue%20state%3Aclosed) (completed in Phase 1)
```